Assets/Project/Global/Events/EventBus.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections.Generic;
+
+public static class EventBus
+{
+    private static Dictionary<Type, List<Action<object>>> eventHandlers;
+
+    static EventBus()
+    {
+        eventHandlers = new Dictionary<Type, List<Action<object>>>();
+    }
+
+    public static void Subscribe<T>(Action<T> handler)
```

Comment:
Suggestion: use constrain to easily say what kind of type you will return. Also, it would be good to have instead the pure type an Interface so you constrain what kind of handlers you want to handle. hehe. BTW nice implementation easy and straight forward